packages/orbit-tailwind-preset/src/presets/components/index.ts

```diff
@@ -133,6 +144,12 @@ const cfg = (options?: Options): Config => {
           "country-flag-small": defaultTokens.countryFlagSmallWidth,
           "country-flag-medium": defaultTokens.countryFlagMediumWidth,
         },
+        minWidth: {
+          "icon-small": defaultTokens.widthIconSmall,
```

Comment:
those tokens are deprecated. We deprecated all tokens starting with width/height. Instead should be used: 
```
widthIconMedium -> iconMediumSize
widthIconLarge -> iconLargeSize
```

and so on, if there will be a mistake it should be fixed and added appropriate token in the same way as for others :)